riot/riot.py

```diff
@@ -1081,6 +1084,19 @@ def run_cmd_venv(
         env["VIRTUAL_ENV"] = abs_venv
         env["PATH"] = f"{abs_venv}/bin:" + env.get("PATH", "")
 
+        try:
+            sitepkgs_path = (
+                next((Path(abs_venv) / "lib").glob("python*")) / "site-packages"
```

Comment:
Why do we need to call `next` here. Can we just use the first match?

Reply:
Here `next` is indeed getting the first (and hopefully only!) result from the iterator returned by `glob`